src/Kmd.Logic.DocumentGeneration.Client/Kmd.Logic.DocumentGeneration.Client.csproj

```diff
@@ -2,7 +2,9 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <PackageVersion>0.2.0</PackageVersion>
+    <!-- <PackageVersion>0.2.1</PackageVersion> -->
+    <VersionPrefix>0.3.1</VersionPrefix>
```

Comment:
Why add this here in addition to directory.build.props?

Reply:
You're right, it shouldn't be needed.
I found locally that I needed it.
(But it could be that I still had PackageVersion set!)